airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/templates/test_report.html.j2

```diff
@@ -162,7 +162,11 @@ function copyToClipBoard(htmlElement) {
     {% if step_result.status == StepStatus.SUCCESS %}
       <label for="{{ step_result.step.title }}" class="lbl-toggle success">{{ step_result.step.title }} | {{ format_duration(step_result.step.run_duration) }}</label>
     {% elif step_result.status == StepStatus.FAILURE %}
+      {% if not step_result.consider_in_overall_status %}
+      <label for="{{ step_result.step.title }}" class="lbl-toggle failure">{{ step_result.step.title }} | Ignored | {{ format_duration(step_result.step.run_duration) }}</label>
```

Comment:
```suggestion
      <label for="{{ step_result.step.title }}" class="lbl-toggle failure">{{ step_result.step.title }} | Ignored (Failed) | {{ format_duration(step_result.step.run_duration) }}</label>
```

or something to indicate that we won't ignore it if it succeeds (and point out that it did fail although it was ignored), would be helpful, I think